src/components/images/EditImage.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import LocalApi from "../../apis/LocalApi";
+import { Form, Button, Container } from "react-bootstrap";
+
+class EditImage extends Component {
+  state = {
+    selectedFile: null,
+    lot: 0,
+    unitNumber: "",
+    productDescription: ""
+  };
+
+  //When loading this page, retrive the data for this particular apartment from backend
+  componentDidMount() {
+    LocalApi.get(`${process.env.REACT_APP_BASEURL}/images/`)
+      .then(response => {
+        for (let i = 0; i < response.data.length; i++) {
+          if (response.data[i]._id === this.props.match.params.id) {

```

Comment:
I didn't know you could get query params from props, nice! I went with that awkward hook: "let { id } = useParams();"